src/Domain/Common/EntityResult.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Codidact.Core.Domain.Common
+{
+    public class EntityResult<T>
+    {
+        public List<string> Errors { get; set; } = new List<string>();
+
+        public bool Success { get; set; }
+
+        public long Id { get; set; }
+
+        public EntityResult()
+        {
+
+        }
+        public EntityResult(bool success)
+        {
+            Success = success;
```

Comment:
This will set the `Id` field to zero, is this intended? Maybe you meant `long? Id` above?

Reply:
I'll do long? id instead, its more correct